vendor/modules.txt

```diff
@@ -593,7 +593,7 @@ k8s.io/apimachinery/third_party/forked/golang/reflect
 k8s.io/cli-runtime/pkg/genericclioptions
 k8s.io/cli-runtime/pkg/printers
 k8s.io/cli-runtime/pkg/resource
-# k8s.io/client-go v0.26.1 => k8s.io/client-go v0.25.4
+# k8s.io/client-go v0.26.3 => k8s.io/client-go v0.25.4
```

Comment:
why golcient version is  changed?

Reply:
no specific requirement, just a regular update with go mod tidy